forge/lib/tiler.py

```diff
@@ -245,6 +287,111 @@ def create(self):
         logger.info('It took %s to create %s tiles (%s were skipped)' % (
             str(datetime.timedelta(seconds=tend - self.t0)), tilecount.value, skipcount.value))
 
+    # Create AWS sqs queue with all the tiles to create
+    # based on current configuration as well as meta data
+    def createQueue(self):
+        queueName = self.tmsConfig.get('General', 'sqsqueue')
+        maxChunks = int(self.tmsConfig.get('General', 'maxChunks'))
+        self.t0 = time.time()
+        if len(queueName) <= 0:
+            logger.error('Missing queueName')
+            return
+        try:
+            sqs = getSQS()
+            # queue with default message visiblity time of 600. So each message is blocked
+            # for other users for 600 seconds. Default would be 30 seconds. It's that high
+            # because each message contains maxChunks tiles
+            q = sqs.create_queue(queueName, visibility_timeout=600)
```

Comment:
Here, nothing happens. The queue is just returned.

But below (line 311) I check if there is something already in the queue. If yes, it's not created. So we assure that we don't overwrite an existing queue.